funcx_sdk/funcx/sdk/executor.py

```diff
@@ -161,21 +161,21 @@ def submit(self, function, *args, endpoint_id=None, container_uuid=None, **kwarg
                 self._function_registry[function] = function_id
                 log.debug(f"Function registered with id:{function_id}")
 
-        task_id = self._task_counter
-        self._task_counter += 1
+        future_id = self._future_counter
+        self._future_counter += 1
```

Comment:
Hmm; back to our discussion in Slack, suggest instead:
```python
submission_no = self._future_counter
```

Reply:
once it's left the counter, it's an ID (just allocated in sequence) and I think it's nice to preserve that fact in the name